app/templates/_package.json

```diff
@@ -1,30 +1,48 @@
 {
-  "name": "<%= appname %>",
-  "version": "0.0.0",
+  "name": "<%= generatorName %>",
+  "version": "0.0.1",
   "description": "Yeoman generator",
-  "license": "MIT",
-  "main": "app/index.js",
-  "repository": "<%= githubUser %>/<%= appname %>",
+  "keywords": [
+    "javascript",
+    "yeoman-generator",
+    "generator",
+    "inception",
+    "init",
+    "create",
+    "boilerplate",
+    "template",
+    "scaffold"
+  ],
   "author": {
-    "name": "<%= realname %>",
-    "email": "<%= email %>",
-    "url": "<%= githubUrl %>"
+    "name": "<%= gitUser.name || gitUser.login %>",
+    "email": "<%= gitUser.email %>",
+    "url": "<%= gitUser.htmlUrl %>"
   },
+  "main": "app/index.js",
+  "repository": "<%= gitUser.login %>/<%= generatorName %>",
   "scripts": {
-    "test": "mocha"
+    "test": "./node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha -- test/node"
```

Comment:
Why use global version instead of local version already installed via `package.json`? 
Global version can have a different version define in `package.json` and we need to add them on Travis processus 'before_script'